addons/web/static/src/core/tooltip/tooltip_service.js

```diff
@@ -167,6 +167,17 @@ export const tooltipService = {
                 cleanup();
             }
         }
+        function onKeydown(ev){
+            if (ev.key === 'Enter') {
+                let focusedElement = document.activeElement;
+                if (focusedElement && (focusedElement.matches("[data-tooltip], [data-tooltip-template]"))) {
```

Comment:
i think you can do like this 
` if (ev.key === 'Enter' && document.activeElement?.matches("[data-tooltip], [data-tooltip-template]")) {`